src/views/Welcome.vue

```diff
@@ -5,26 +5,26 @@
         <div class="d-flex flex-no-wrap justify-space-around align-center">
           <div class="ml-3"><v-progress-circular size="50" indeterminate color="primary"></v-progress-circular></div> 
           <div>
-            <v-card-text>මේ මෘදුකාංගයේ දත්ත තවමත් copy වෙමින් පවතී. මොහොතක් ඉවසන්න. මෙය පළමු ස්ථාපනය කිරීමේදී පමණක් සිදුවේ.</v-card-text>
+            <v-card-text>මෙම මෘදුකාංගයේ දත්ත පිටපත් වෙමින් පවතී. මොහොතක් රැඳී සිටින්න. මෙය පළමු ස්ථාපනයේ දී පමණක් සිදුවේ.</v-card-text>
           </div>
         </div>
       </v-card>
     </v-dialog>
 
     <v-banner icon="mdi-alert-circle" color="warning">
-      පාලි පෙළ සෝදුපත් තවමත් අවසන් කර නැති බව සලකන්න. අකුරු වැරදි තිබිය හැකිය. පිටු අංක මත ඔබා අදාළ scan පිටුව පරික්ෂා කර බැලිය හැක.
-      අටුවා පරිවර්තනයේද පරිවර්තන දෝෂ, මුද්‍රණ දෝෂ සහ පරිවර්තන මග හැරුණු තැන් තිබිය හැක. දෝෂ හමුවුවහොත් නිවැරදි කිරීමට අප වෙත දන්වා එවන්න. pathnirvana@gmail.com
+      පාලි පෙළ සෝදුපත් කියවීම තවමත් අවසන් කර නැති බව සලකන්න. අකුරු වැරදි තිබීමට ඉඩ ඇත. පිටු අංක මත එබීමෙන් අදාළ පරිලෝකිත පිටුව බැලීමට හැකිය.
+      අටුවා පරිවර්තනයේ ද පරිවර්තන දෝෂ, මුද්‍රණ දෝෂ හා පරිවර්තන මග හැරුණු තැන් තිබීමට හැකිය. දෝෂ හමු වුවහොත් ඒවා නිවැරදි කිරීමට අප වෙත දන්වා එවන්න. pathnirvana@gmail.com
     </v-banner>
 
     <v-container fluid>
       <v-row>
         <v-col cols="12">
           <v-card flat>
-            <v-card-title>ශ්‍රී ලංකා තිපිටක පෙළ, අටුවා සහ සිංහල පරිවර්තනය</v-card-title>
+            <v-card-title>ත්‍රිපිටක පෙළ, අටුවා සහ සිංහල පරිවර්තනය</v-card-title>
```

Comment:
sri lanka must be included since there are different versions of the tipitaka. 
also tipitaka is the correct pali word not tripitaka. Please keep it unchanged as tipitaka.

Reply:
පැහැදිලි කිරීමට ස්තූතියි. වෙනස්කම් සියල්ල ඉවත් කර ඇත.

_තොරතුරු: **(මෙය අපගේ සමූහයේ කාර්යන්ට අදාළ නොවන හා එයින් බැහැර දෙයකි)**
විවිධ නම් වලින් ත්‍රිපිටක/තිපිටක පැවතීම ගැටළුවක් විය හැකි බව අපගේ සමාජිකයින් පැවසීම නිසා ඉහත වෙනස්කම් සිදු කළෙමි. මේ අනුව පුද්ගලයින් තමන්ගේ අභිමතයට අනුව විවිධාකාර තිපිටක ප්‍රකාශයට පත් කිරීමෙන් ධර්මයට හානිවන පසුබිමක් පහසුවෙන් ගොඩනැගෙයි._
_ශ්‍රී ලංකාව යනු මෑත දී භාවිතයට ගත් නමකි. ඉතිහාස තොරතුරු වලට අනුව මෙරට ග්‍රන්ථාරූඩ වූ ත්‍රිපිටකය මත පසුකාලීන ලේඛන පදනම් වන බැවින් සියළුම ප්‍රකාශන, මුද්‍රණ තනි ප්‍රධාන නමක්/අනන්‍යතාවක් යටතේ පවත්වා ගනිමින් අතිරේක අනුවාද/සංස්කරණ නාමයක් ප්‍රධාන නම අභිබවා නොයන ආකාරයෙන් භාවිතා කරන්නේ නම් වඩාත් අගනේය._